src/vl-mapactions-draw-action.js

```diff
@@ -80,8 +92,18 @@ export class VlDrawAction extends VlMapAction {
     this.drawInteraction = drawInteraction;
   }
 
+  activate() {
+    if (this.options.snapping && this.options.snapping.layer) {
+      this.map.addLayer(this.options.snapping.layer);
+    }
```

Comment:
Wanneer `options.snapping.layer` niet gedefinieerd is, wordt default de huidige layer gekozen. De check op `options.snapping.layer` lijkt me daardoor overbodig/fout?

Reply:
Nee, ik zou precies niet 'de laag waarop je tekent' beginnen weghalen bij het activeren en deactiveren van de tekenactie daarop ...

Reply:
Enkel als ge een aparte 'snapping layer' meegeeft, zou ik die op en af de kaart halen.

Reply:
Inderdaad, anders was de kaartlaag al gedefinieerd.